src/ALAC/ag_dec.c

```diff
@@ -26,6 +26,10 @@
 	Copyright:	(c) 2001-2011 Apple, Inc.
 */
 
+#if HAVE_CONFIG_H
+#include "config.h"
+#endif
```

Comment:
Commit 267285f.

    #if HAVE_CONFIG_H
    ...
    #endif

Old stuff. Probably from times, when i thought not to configure `config.h` and pass required defines directly to compiler with `/D` switch.

`#include "config.h` is required here because some old compilers doesn't support `__inline` keyword.  It is VS2013 and before, some old GCC versions.

I guess it can be removed. VS2013 has bad C support anyway. VS2015 is much better and will work without it.